client/layout/style.scss

```
@@ -130,7 +130,16 @@ body.is-focus-sites {
 		padding-bottom: 0;
 	}
 }
-
+body.is-mobile-app-view {
+	/* We are ignoring these lines because without the px value the calc function does not work as expected */
+	/* stylelint-disable */
```

Could we disable only the specific rule?

```suggestion
	/* stylelint-disable length-zero-no-unit */
```